app/views/dashboard/_nav.html.erb

```diff
@@ -0,0 +1,40 @@
+ <!-- Side bar-->
+<div id="sidebar" class="h-screen w-16 menu bg-white text-white px-4 flex items-center nunito static fixed shadow">
+    <ul class="list-reset ">
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/index" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fas fa-home fa-fw mr-3 text-indigo-400"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Home</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0 ">
+            <a href="/dashboard/business_date" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fas fa-tasks fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Datos del negocio</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/providers" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fas fa-truck-loading fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Proveedores</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/inventory" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fas fa-chart-area fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Inventario</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/sales_and_purcharses" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fa-regular fa-pen-to-square fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Registro de ventas y compras</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/statement_of_income" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fa-solid fa-chart-simple fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Estado de resultado</span>
+            </a>
+        </li>
+        <li class="my-2 md:my-0">
+            <a href="/dashboard/balance_sheet" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-indigo-400">
+                <i class="fas fa-balance-scale fa-fw mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Balance general</span>
+            </a>
+        </li>
+    </ul>
+</div>
```

Comment:
Check the indentation of your file (space: 2)

Reply:
Done